engines/router/missionctl/fiberapi/routing_strategy.go

```diff
@@ -82,6 +84,23 @@ func (r *DefaultTuringRoutingStrategy) SelectRoute(
 		}
 	}
 
+	// Use the route name path to locate the name of the route to be returned as the final response
+	if r.routeSelectionPolicy.routeNamePath != "" {
+		routeName, err := jsonparser.GetString(
+			experimentResponse.Body(),
+			strings.Split(r.routeSelectionPolicy.routeNamePath, ".")...,
+		)
+
+		if err != nil {
+			log.WithContext(ctx).Errorf(err.Error())
+			return nil, fallbacks, nil
+		}
+
+		if selectedRoute, ok := routes[routeName]; ok {
+			return selectedRoute, []fiber.Component{}, nil
```

Comment:
If selectedRoute is not found, shall we also log an error here like the above block, so the user can see the error logs for the router?

Reply:
Okay, just added that :)